assets/javascripts/discourse/templates/list/topic-list-item.raw.hbs

```diff
@@ -15,22 +15,39 @@
 
   <div class="topic-details">
     {{#unless tilesOrSocial}}
-      {{raw "list/topic-list-title" topic=topic showTopicPostBadges=showTopicPostBadges}}
-      {{#if showCategoryBadge}}
-        {{raw "list/topic-category" category=topic.category}}
-      {{/if}}
-      {{discourse-tags topic mode="list"}}
+    <span class='link-top-line'>
```

Comment:
I'm just duplicating what is in the core template ... better to match as closely as possible to core, no?

Reply:
@merefield We've already gone with a seperate topic-list-title template for various reasons, so we unless we get rid of that template (which we shouldn't), we need to keep topic title logic there, otherwise we'll have too much duplication.

Reply:
ok no problem, i'll migrate

Reply:
Done.  Apologies I now see the inconsistency I left, doh!